matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/terms/DefaultTermsService.kt

```diff
@@ -63,12 +63,12 @@ internal class DefaultTermsService @Inject constructor(
                                       baseUrl: String,
                                       agreedUrls: List<String>,
                                       token: String?) {
-        withContext(coroutineDispatchers.main) {
+        withContext(coroutineDispatchers.io) {
```

Comment:
I think this `withContext` can be removed.

Reply:
Done, thanks